packages/components/src/radio-control/README.md

```diff
@@ -104,7 +104,8 @@ A function that receives the value of the new option that is being selected as i
 
 An array of objects containing the value and label of the options.
 
--   `label`: `string` The label to be shown to the user.
+-   `label`: `ReactNode` The label to be shown to the user. When the label is not a string, make sure that the element is accessibly labeled.
```

Comment:
Matching TypeScript types

```suggestion
-   `label`: `string` | `React.ReactElement` The label to be shown to the user. When the label is not a string, make sure that the element is accessibly labeled.
```

Reply:
Based on https://github.com/WordPress/gutenberg/pull/66615#discussion_r1861323374, it looks like we may want to revert to `ReactNode`. In that case, feel free to ignore my suggestion above, and instead let's change the TypeScript type back to `ReactNode`.